common/persistence/sql/sqlplugin/definition.go

```diff
@@ -38,6 +38,9 @@ type (
 	ClusterMetadataRow struct {
 		ImmutableData         []byte
 		ImmutableDataEncoding string
```

Comment:
according to the PR (schema changes?) these 2 field should be removed?

Reply:
Leftover, cleaned up.